cargo-smart-release/src/crates_index.rs

```diff
@@ -36,7 +36,7 @@ fn default_path() -> PathBuf {
         .or_else(|| {
             std::env::var_os("CARGO_HOME")
                 .map(PathBuf::from)
-                .or_else(|| std::env::var_os("HOME").map(|dir| Path::new(&dir).join(".cargo")))
+                .or_else(|| home::home_dir().map(|dir| Path::new(&dir).join(".cargo")))
```

Comment:
this is different because cargo-smart-release is not a git tool so I used `home::home_dir` here (which is already a dependency for `cargo-smart-release`). All other places use `gix_path::home`